src/style-spec/reference/v8.json

```diff
@@ -652,7 +652,7 @@
     },
     "filter": {
       "type": "filter",
-      "doc": "A expression specifying conditions on source features. Only features that match the filter are displayed. Zoom expressions in filters are only evaluated at integer zoom levels. The `feature-state` expression is not supported in filter expressions."
+      "doc": "A expression specifying conditions on source features. Only features that match the filter are displayed. Zoom expressions in filters are only evaluated at integer zoom levels. The `feature-state` expression is not supported in filter expressions.  The `['pitch']` and `['distance-from-center']` expressions are supported only for filter expressions on the `symbol` layer."
```

Comment:
Drive-by fix!

```suggestion
      "doc": "An expression specifying conditions on source features. Only features that match the filter are displayed. Zoom expressions in filters are only evaluated at integer zoom levels. The `feature-state` expression is not supported in filter expressions.  The `['pitch']` and `['distance-from-center']` expressions are supported only for filter expressions on the `symbol` layer."
```

Reply:
Looks good, but it's getting to be quite a few very precise conditions in quick succession, which makes it a little challenging to parse. I think it's okay for now, but any more details will probably either require splitting this into a couple paragraphs or adding some more filler words or context to make it flow a little more smoothly.